plugins/Microsoft.DocAsCode.Build.UniversalReference/BuildInputs/SyntaxDetailViewModel.cs

```diff
@@ -36,14 +36,15 @@ public class SyntaxDetailViewModel
         /// <summary>
         /// syntax's returns
         /// multiple return type is allowed in languages like JavaScript, Python
+        /// ApiParameter supports multiple types
         /// </summary>
-        [YamlMember(Alias = Constants.PropertyName.Returns)]
-        [JsonProperty(Constants.PropertyName.Returns)]
-        public List<ApiParameter> Returns { get; set; }
+        [YamlMember(Alias = Constants.PropertyName.Return)]
+        [JsonProperty(Constants.PropertyName.Return)]
+        public ApiParameter Return { get; set; }
```

Comment:
ApiParameter's type is a list

---
In reply to: [114244536](https://github.com/dotnet/docfx/pull/1618#discussion_r114244536) [](ancestors = 114244536)